### PR TITLE
Doc update as per comments from Robin

### PR DESCRIPTION
**Pull request by dhiraj0116**

### CODE REVIEW

File: src/main/java/io/cdap/plugin/ariba/source/config/AribaPluginConfig.java [hunk 1]
```diff
@@ -63,11 +63,11 @@ public class AribaPluginConfig extends ReferencePluginConfig {
   private final String systemType;
 
   @Macro
-  @Description("Realm name for the customer.")
+  @Description("Target Realm name from which data is to be extracted.")
   private final String realm;
 
   @Macro
-  @Description("Name of the view template you wish to create.")
+  @Description("Name of the View Template from which data to be extracted.")
```
Comment (rrielley76) [inline src/main/java/io/cdap/plugin/ariba/source/config/AribaPluginConfig.java]: add "is". View Template from which data is to be extracted.

File: widgets/Ariba-batchsource.json [hunk 2]
```diff
@@ -47,15 +47,15 @@
...
+            "placeholder": "Target Realm name from which data is to be extracted. E.g. : S4SVAll"
           }
         },
         {
           "widget-type": "textbox",
           "label": "View Template Name",
           "name": "viewTemplateName",
           "widget-attributes": {
-            "placeholder": "Specify the name of the view template you wish to create. E.g. : RequestFactSystemView"
+            "placeholder": "Name of the View Template from which data to be extracted. E.g. : RequestFactSystemView"
```
Comment (rrielley76) [inline widgets/Ariba-batchsource.json]: add "is". Name of the View Template from which data is to be extracted.

File: src/main/java/io/cdap/plugin/ariba/source/config/AribaPluginConfig.java (hunk 1 above)
Comment (dhiraj0116) [reply src/main/java/io/cdap/plugin/ariba/source/config/AribaPluginConfig.java]: Added

File: widgets/Ariba-batchsource.json (hunk 2 above)
Comment (dhiraj0116) [reply widgets/Ariba-batchsource.json]: Added

File: src/main/java/io/cdap/plugin/ariba/source/config/AribaPluginConfig.java [hunk 3]
```diff
@@ -63,11 +63,11 @@ public class AribaPluginConfig extends ReferencePluginConfig {
   private final String systemType;
 
   @Macro
-  @Description("Realm name for the customer.")
+  @Description("Target Realm name from which data is to be extracted.")
```
Comment (rrielley76) [inline src/main/java/io/cdap/plugin/ariba/source/config/AribaPluginConfig.java]: Don't think you need Target here. How about "Realm name from which data is to be extracted."

File: src/main/java/io/cdap/plugin/ariba/source/config/AribaPluginConfig.java [hunk 4]
```diff
@@ -63,11 +63,11 @@ public class AribaPluginConfig extends ReferencePluginConfig {
...
   private final String realm;
 
   @Macro
-  @Description("Name of the view template you wish to create.")
+  @Description("Name of the View Template from which data is to be extracted.")
```
Comment (rrielley76) [inline src/main/java/io/cdap/plugin/ariba/source/config/AribaPluginConfig.java]: View Template should be lowercase, like in Ariba docs/blogs. "view template"

File: widgets/Ariba-batchsource.json [hunk 5]
```diff
@@ -47,15 +47,15 @@
           "label": "Realm",
           "name": "realm",
           "widget-attributes": {
-            "placeholder": "Specify the realm name for the customer. E.g. : S4SVAll"
+            "placeholder": "Target Realm name from which data is to be extracted. E.g. : S4SVAll"
```
Comment (rrielley76) [inline widgets/Ariba-batchsource.json]: Remove "Target" and spell out for example. "Realm name from which the data is to be extracted, for example, S4VALL."

File: widgets/Ariba-batchsource.json [hunk 6]
```diff
@@ -47,15 +47,15 @@
...
           }
         },
         {
           "widget-type": "textbox",
           "label": "View Template Name",
           "name": "viewTemplateName",
           "widget-attributes": {
-            "placeholder": "Specify the name of the view template you wish to create. E.g. : RequestFactSystemView"
+            "placeholder": "Name of the View Template from which data is to be extracted. E.g. : RequestFactSystemView"
```
Comment (rrielley76) [inline widgets/Ariba-batchsource.json]: Change to "Name of the view template from which data is to be extracted, for example, RequestFactSystemView."

File: src/main/java/io/cdap/plugin/ariba/source/config/AribaPluginConfig.java (hunk 3 above)
Comment (dhiraj0116) [reply src/main/java/io/cdap/plugin/ariba/source/config/AribaPluginConfig.java]: Added

File: src/main/java/io/cdap/plugin/ariba/source/config/AribaPluginConfig.java (hunk 4 above)
Comment (dhiraj0116) [reply src/main/java/io/cdap/plugin/ariba/source/config/AribaPluginConfig.java]: Added

File: widgets/Ariba-batchsource.json (hunk 5 above)
Comment (dhiraj0116) [reply widgets/Ariba-batchsource.json]: Added

File: widgets/Ariba-batchsource.json (hunk 6 above)
Comment (dhiraj0116) [reply widgets/Ariba-batchsource.json]: Added

File: widgets/Ariba-batchsource.json [hunk 7]
```diff
@@ -97,15 +97,15 @@
           "label": "From Date",
           "name": "fromDate",
           "widget-attributes": {
-            "placeholder": "Start date of the extract. E.g. 2021-12-01T00:00:00Z"
+            "placeholder": "Start date of the extract, for example, 2021-12-01T00:00:00Z"
```
Comment (rrielley76) [inline widgets/Ariba-batchsource.json]: Can we change extract to extraction?

File: widgets/Ariba-batchsource.json [hunk 8]
```diff
@@ -97,15 +97,15 @@
...
           }
         },
         {
           "widget-type": "textbox",
           "label": "To Date",
           "name": "toDate",
           "widget-attributes": {
-            "placeholder": "End date of the extract. E.g. 2022-03-29T00:00:00Z"
+            "placeholder": "End date of the extract, for example, 2022-03-29T00:00:00Z"
```
Comment (rrielley76) [inline widgets/Ariba-batchsource.json]: Can we change extract to extraction?

Comment (dhiraj0116) [reply widgets/Ariba-batchsource.json]: Changed

File: widgets/Ariba-batchsource.json (hunk 7 above)
Comment (dhiraj0116) [reply widgets/Ariba-batchsource.json]: Changed

File: docs/Ariba-batchsource.md [hunk 9]
```diff
@@ -23,21 +23,21 @@ You can configure the following properties for the Ariba.
 
 **Reference Name:** Name used to uniquely identify this source for lineage,
 annotating metadata, etc.  
-**API Endpoint (M)**: Ariba Base URL.  
-**System Type**: Select system either Prod or Sandbox.  
-**Realm (M)**: Realm name of the customer.   
-**View Template Name (M)**: Name of the Template.
+**API Endpoint (M)**: Specify the base Path of Ariba API, for example, https://sandbox.api.sap.com.
```
Comment (rrielley76) [inline docs/Ariba-batchsource.md]: We usually just give the definition: "Base path of the Ariba API, for example, https://sandbox.api.sap.com."

File: docs/Ariba-batchsource.md [hunk 10]
```diff
@@ -23,21 +23,21 @@ You can configure the following properties for the Ariba.
 
 **Reference Name:** Name used to uniquely identify this source for lineage,
 annotating metadata, etc.  
-**API Endpoint (M)**: Ariba Base URL.  
-**System Type**: Select system either Prod or Sandbox.  
-**Realm (M)**: Realm name of the customer.   
-**View Template Name (M)**: Name of the Template.
+**API Endpoint (M)**: Specify the base Path of Ariba API, for example, https://sandbox.api.sap.com.  
+**System Type**: Select System either prod/sandbox.
```
Comment (rrielley76) [inline docs/Ariba-batchsource.md]: Change to: "Type of system the Ariba instance is running on: Production or Sandbox." (should we change the label Prod to Production?

File: widgets/Ariba-batchsource.json [hunk 11]
```diff
@@ -20,7 +20,7 @@
           "label": "API Endpoint",
           "name": "baseURL",
           "widget-attributes": {
-            "placeholder": "Specify the base Path of Ariba API.  E.g. : https://sandbox.api.sap.com"
+            "placeholder": "Specify the base Path of Ariba API, for example, https://sandbox.api.sap.com"
```
Comment (rrielley76) [inline widgets/Ariba-batchsource.json]: Change to "Base path of the Ariba API, for example, https://sandbox.api.sap.com."

File: widgets/Ariba-batchsource.json [hunk 12]
```diff
@@ -68,15 +68,15 @@
...
+            "placeholder": "Ariba Client ID"
           }
         },
         {
           "widget-type": "password",
           "label": "Ariba Client Secret",
           "name": "clientSecret",
           "widget-attributes": {
-            "placeholder": "Client Secret"
+            "placeholder": "Ariba Client Secret"
```
Comment (rrielley76) [inline widgets/Ariba-batchsource.json:79]: We need to add Ariba before API Key tooltip too....

File: docs/Ariba-batchsource.md (hunk 9 above)
Comment (dhiraj0116) [reply docs/Ariba-batchsource.md]: Changed

File: docs/Ariba-batchsource.md (hunk 10 above)
Comment (dhiraj0116) [reply docs/Ariba-batchsource.md]: Added

File: widgets/Ariba-batchsource.json (hunk 11 above)
Comment (dhiraj0116) [reply widgets/Ariba-batchsource.json]: Changed

File: widgets/Ariba-batchsource.json (hunk 12 above)
Comment (dhiraj0116) [reply widgets/Ariba-batchsource.json:79]: Already added

File: docs/Ariba-batchsource.md (hunk 10 above)
Comment (dhiraj0116) [reply docs/Ariba-batchsource.md]: Also changed the label from Prod to Production

**Review by rrielley76 — APPROVED** [review]

**Pull request merged**